tests/test_nwm22_grid_loading.py

```diff
@@ -0,0 +1,65 @@
+
+from pathlib import Path
+import filecmp
+
+import pandas as pd
+
+from teehr.loading.nwm22.nwm_grid_data import fetch_and_format_nwm_grids
+from teehr.loading.nwm_common.utils_nwm import build_zarr_references
+from teehr.loading.nwm22.const_nwm import (
+    NWM22_UNIT_LOOKUP,
+)
+
+TEST_DIR = Path("tests", "data", "nwm22")
+TEMP_DIR = Path("tests", "data", "temp")
+WEIGHTS_FILEPATH = Path(TEST_DIR, "onehuc10_weights.parquet")
+
+
+def test_grid_zarr_reference_file():
+
+    component_paths = [
+        "gcs://national-water-model/nwm.20201218/forcing_analysis_assim/nwm.t00z.analysis_assim.forcing.tm00.conus.nc" # noqa
+    ]
+
+    _ = build_zarr_references(
+        remote_paths=component_paths,
+        json_dir=TEMP_DIR,
+        ignore_missing_file=False
+    )
+
+    json_file = Path(TEMP_DIR,
```

Comment:
@samlamont would it make sense to reference the response from `build_zarr_references` instead of hard coding?

Reply:
Yes, not sure why I hard-coded the path for `json_file`